doc/quickstart/interfaces.rst

```diff
@@ -0,0 +1,17 @@
+.. _intro_interfaces:
+
+Interfaces
+==========
+
+PennyLane integrates quantum nodes with other programming and machine learning frameworks.
+Such frameworks are called *interfaces*. The default interface, implicitly used in the previous
+:ref:`Introduction <pl_intro>`, is NumPy.
```

Comment:
The default interface, implicitly used in the :ref:`Introduction <pl_intro>`, is NumPy.